README.md

```diff
@@ -56,6 +56,20 @@ The first command will build the source of your application. The second command
 
 You can find your API Gateway Endpoint URL in the output values displayed after deployment.
 
+### Deploy into VPC
+
+If the application is required to be deployed in private VPC(VPCs), please add following configuration to `AWS::Serverless::Api` resource:
```

Comment:
Let's include more text about the need to create the VPC endpoints separately, either manually via console, separate CFTs, or as part of the SAM template.

Reply:
Done